File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.py
import sys
def print(*args, sep=" ", end="\n"): return sys.stdout.write(sep.join(map(str, args)) + end)
def input(): return sys.stdin.readline().rstrip()

def main():
    for j in range(3):
        s = input()
        if s.isdigit():
            i = int(s)+3-j
            if i % 3 == 0 and i % 5 == 0: print('FizzBuzz')
            elif i % 3 == 0 and i % 5 != 0: print('Fizz')
            elif i % 3 != 0 and i % 5 == 0: print('Buzz')
            else: print(i)
            break 
    return 0
if __name__ == '__main__':
    sys.exit(main())

File: 백준/Bronze/28702. FizzBuzz/FizzBuzz.cc
#include<bits/stdc++.h>
using namespace std;
using ll = long long;

int main(){
    cin.tie(0); cout.tie(0); ios::sync_with_stdio(false);
    for(int j = 0; j < 3; j++){
        string s; cin >> s;
        if(isdigit(s[0])){
            int i = stoi(s)+3-j;
            if(i % 3 == 0 && i % 5 == 0) cout << "FizzBuzz";
            else if(i % 3 == 0 && i % 5 != 0) cout << "Fizz";
            else if(i % 3 != 0 && i % 5 == 0) cout << "Buzz";
            else cout << i;
            break;
        }
    }
    return 0;
}